### PR TITLE
Directive split

### PR DESCRIPTION
**Pull request by akhmorgan**
Splits index.html down a bit.

### CODE REVIEW

File: src/sidebars/sidebar.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="sidebars" ng-controller="SidebarCtrl as sidebarCtrl">
+    <div class="favourites {{sidebarCtrl.favouritesClass()}}" ng-controller="FavouritesCtrl as favouritesCtrl" ng-click="sidebarCtrl.favouritesClick()">
```
Comment (owennw) [inline src/sidebars/sidebar.html]: Given the controller is declared in the directive does it need to be (re)declared here?

File: src/toolbar/toolbar.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div id="toolbar" ng-controller="ToolbarCtrl as toolbarCtrl">
+    <img id="logo" src="../assets/svg/scottlogic_logo.svg" />
```
Comment (owennw) [inline src/toolbar/toolbar.html]: Same comment as above. I'm also curious about the relative path here too.

**Comment by owennw** [comment]
Couple of small comments, looks good though!

File: src/sidebars/sidebar.html (hunk 1 above)
Comment (akhmorgan) [inline src/sidebars/sidebar.html]: You're quite right. I'll fix that.

**Pull request merged**